### PR TITLE
Fix mismatch fluentbitconfig

### PR DESCRIPTION
**Pull request by benjaminhuo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,6 +229,29 @@ Within a couple of minutes, you can double check the results in the Elasticsearc
...
+    hostPath:
+      path: /var/lib/fluent-bit/
+  resources:
+    requests:
+      cpu: 10m
+      memory: 25Mi
+    limits:
+      cpu: 500m
+      memory: 200Mi
+  fluentBitConfigName: fluent-bit-config
```
Comment (zhu733756) [inline README.md:250]: => fluent-bit-only-config?

Comment (wenchajun) [reply README.md:250]: This may require modifying the fluentbit Conifg name as well

File: README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ spec:
     limits:
       cpu: 500m
       memory: 200Mi
-  fluentBitConfigName: fluent-bit-config
+  fluentBitConfigName: fluent-bit-only-config
   tolerations:
```
Comment (zhu733756) [inline README.md:83]: Yes, here changed.

**Comment by benjaminhuo** [comment]
This is a fix for https://fluent-all.slack.com/archives/C02642TE9HR/p1648843305447079

**Comment by benjaminhuo** [comment]
fluent-bit-only-config is for the fluentbit-only case
fluent-bit-config is for the fluentbit + fluentd case
These two cases need two different config names
@zhu733756 @wenchajun

**Comment by zhu733756** [comment]
Yes. you're right. The commits cannot see the whole docs that lead us a mistake.

**Pull request merged**